lib/elixir/lib/kernel.ex

```diff
@@ -3401,6 +3401,45 @@ defmodule Kernel do
   In the example above, a `sum/2` function is defined; this function receives
   two arguments and returns their sum.
 
+  ## Default arguments
+
+  `\\` is used to specify a default value for a parameter of a function. For
+  example:
```

Comment:
Could we start a new line with . "For example:"
No blank line in between and markdown will render them as one paragraph.
This gives us more room to edit sentences in the future, without affecting Git blame